### ISSUE TITLE
Accessibility - Zebra tooltips do not react to focus events

### ISSUE THREAD

**Issue by xmlns** [stefangabos/Zebra_Tooltips#39]
I added and tested functionality for Zebra Tooltips a while ago, I will be a submitting a PR for this shortly.
Tooltips should be accessible with the especially if the tabindex attribute is used to manually add them in the browser's keyboard navigation.

**Comment by stefangabos** [comment]
i agree
